mailbox/cassandra/pom.xml

```diff
@@ -62,6 +62,16 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>blob-cassandra</artifactId>
```

Comment:
That, should be with scope test. We do not want to enforce the cassandra blob store choice, do we?